Framework/include/QualityControl/PostProcessingRunner.h

```diff
@@ -49,6 +49,8 @@ class PostProcessingRunner
   void stop();
   /// \brief Reset transition. Throws on errors.
   void reset();
+  /// \brief Runs the task over selected timestamps, performing the full start, run, stop cycle.
```

Comment:
could you add a line about the expected argument ?

Reply:
done

Reply:
ty